avro2py/utils.py

```diff
@@ -53,8 +53,11 @@ def _safe_convert_namedtuple(v: Union[NamedTuple, Any]) -> Union[AvroDictType, A
 
 @_CONVERTERS_TO_AVRO.append
 def _safe_convert_list(v: Union[List, Any]) -> Union[AvroDictType, Any]:
-    if isinstance(v, list) and len(v) and hasattr(v[0], '_asdict'):
-        return [to_avro_dict(ele) for ele in v]
+    if isinstance(v, list) and len(v):
+        if hasattr(v[0], '_asdict'):
+            return [to_avro_dict(ele) for ele in v]
+        else:
+            return [_convert_types_to_avro(ele) for ele in v]
```

Comment:
Output of to_avro_dict was following

Before the code change
```
{'tags': [<Status.ACTIVE: 'Active'>, <Status.PAUSED: 'Paused'>]} # List[Enum]
```

After the code change
```
{'tags': ['Active', 'Paused']}. # List of Str
```